### PR TITLE
Add set password email sending to new users

### PR DESCRIPTION
**Pull request by MorayM**

### CODE REVIEW

File: GLAA.Services/Admin/AdminUserPostDataHandler.cs [hunk 1]
```diff
@@ -32,6 +39,20 @@ public string Insert(AdminUserViewModel model)
 
             userManager.AddToRoleAsync(user, model.Role).GetAwaiter().GetResult();
 
+            // For more information on how to enable account confirmation and password reset please
+            // visit https://go.microsoft.com/fwlink/?LinkID=532713
+            var code = userManager.GeneratePasswordResetTokenAsync(user).GetAwaiter().GetResult();
```
Comment (dougajmcdonald) [inline GLAA.Services/Admin/AdminUserPostDataHandler.cs:44]: We definitely need to look at this in more detail, at some point, 99.9% sure the async stuff _should_ work.

File: GLAA.Services/Admin/AdminUserPostDataHandler.cs [hunk 2]
```diff
@@ -32,6 +39,20 @@ public string Insert(AdminUserViewModel model)
...
+            var callbackUrl = url.Action("ResetPassword", "AccountController", new { userId = user.Id, code = code }, scheme);
```
Comment (dougajmcdonald) [inline GLAA.Services/Admin/AdminUserPostDataHandler.cs:45]: This is cool, didn't know you could set a callbackUrl like this

**Review by dougajmcdonald — APPROVED** [review]
Looks cool, I think we should look at the async bits again at some point though

File: GLAA.Services/Admin/AdminUserPostDataHandler.cs (hunk 1 above)
Comment (MorayM) [reply GLAA.Services/Admin/AdminUserPostDataHandler.cs:44]: Yeah, it just isn't working. As I understand it should automatically wait for an `await`ed operation to complete when the result is needed but it doesn't.

File: GLAA.Services/Admin/AdminUserPostDataHandler.cs (hunk 2 above)
Comment (MorayM) [reply GLAA.Services/Admin/AdminUserPostDataHandler.cs:45]: Open source frameworks are great!

**Pull request merged**